src/models/contact.js

```diff
@@ -132,6 +132,39 @@ WebPage.attributes = {
   }),
 };
 
+class Groups extends RestfulModel {
+  toJSON() {
```

Comment:
np: I think if you define `jsonKey` for all the Groups' attributes below, inheriting the parent RestfulModel's toJSON() will work without defining one here.

Reply:
I tried to change this but the tests kept failing. I'm confused on where this `toJSON()` thing is coming from and how it is working so I'm just going to keep it as is. I

Reply:
Got clarity from Maria and have updated this!